--- s3transfer-0.3.4/tests/functional/test_manager.py.orig	2021-01-11 20:17:48.000000000 +0100
+++ s3transfer-0.3.4/tests/functional/test_manager.py	2021-02-14 10:53:55.517004397 +0100
@@ -12,7 +12,10 @@
 # language governing permissions and limitations under the License.
 from io import RawIOBase
 from botocore.awsrequest import create_request_object
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 
 from tests import skip_if_using_serial_implementation
 from tests import StubbedClientTest
--- s3transfer-0.3.4/tests/functional/test_processpool.py.orig	2021-01-11 20:17:48.000000000 +0100
+++ s3transfer-0.3.4/tests/functional/test_processpool.py	2021-02-14 10:54:07.833604339 +0100
@@ -14,7 +14,10 @@ import glob
 import os
 from multiprocessing.managers import BaseManager
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 import botocore.exceptions
 import botocore.session
 from botocore.stub import Stubber
--- s3transfer-0.3.4/tests/functional/test_upload.py.orig	2021-01-11 20:17:48.000000000 +0100
+++ s3transfer-0.3.4/tests/functional/test_upload.py	2021-02-14 10:53:48.667041507 +0100
@@ -15,7 +15,10 @@ import time
 import tempfile
 import shutil
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 from botocore.client import Config
 from botocore.exceptions import ClientError
 from botocore.awsrequest import AWSRequest
--- s3transfer-0.3.4/tests/unit/test_bandwidth.py.orig	2021-01-11 20:17:48.000000000 +0100
+++ s3transfer-0.3.4/tests/unit/test_bandwidth.py	2021-02-14 10:54:14.116903633 +0100
@@ -14,7 +14,10 @@ import os
 import shutil
 import tempfile
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 
 from tests import unittest
 from s3transfer.bandwidth import RequestExceededException
--- s3transfer-0.3.4/tests/unit/test_download.py.orig	2021-01-11 20:17:48.000000000 +0100
+++ s3transfer-0.3.4/tests/unit/test_download.py	2021-02-14 10:54:24.266848646 +0100
@@ -15,7 +15,10 @@ import os
 import shutil
 import tempfile
 import socket
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 
 from tests import BaseTaskTest
 from tests import BaseSubmissionTaskTest
--- s3transfer-0.3.4/tests/unit/test_futures.py.orig	2021-01-11 20:17:48.000000000 +0100
+++ s3transfer-0.3.4/tests/unit/test_futures.py	2021-02-14 10:55:05.729957355 +0100
@@ -14,7 +14,10 @@ import sys
 import time
 import traceback
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 from concurrent.futures import ThreadPoolExecutor
 
 from tests import unittest
--- s3transfer-0.3.4/tests/unit/test_processpool.py.orig	2021-01-11 20:17:48.000000000 +0100
+++ s3transfer-0.3.4/tests/unit/test_processpool.py	2021-02-14 10:54:35.633453735 +0100
@@ -15,7 +15,10 @@ import signal
 import time
 import threading
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 from six.moves import queue
 from botocore.exceptions import ClientError
 from botocore.exceptions import ReadTimeoutError
--- s3transfer-0.3.4/tests/unit/test_s3transfer.py.orig	2021-01-11 20:17:48.000000000 +0100
+++ s3transfer-0.3.4/tests/unit/test_s3transfer.py	2021-02-14 10:54:29.480153736 +0100
@@ -17,7 +17,10 @@ import socket
 from tests import unittest
 from contextlib import closing
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 from botocore.vendored import six
 from concurrent import futures
 
--- s3transfer-0.3.4/tests/unit/test_utils.py.orig	2021-01-11 20:17:48.000000000 +0100
+++ s3transfer-0.3.4/tests/unit/test_utils.py	2021-02-14 10:54:55.406679947 +0100
@@ -19,7 +19,10 @@ import re
 import time
 import io
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 
 from tests import unittest
 from tests import RecordingSubscriber
